schemas/constants.js

```diff
@@ -135,4 +135,13 @@ module.exports = {
 
   // Pattern for scope names, for when-ever that is useful
   "scope-pattern":   "^[\\x20-\\x7e]*$",
+
+  // Maximum number of dependencies a single task can have
+  "max-task-dependencies": 100,
```

Comment:
It was a prettt round number :)

Obviously there is some limit, I don't know what it is... But 100 should serve most use-cases and force intermediate dummy tasks in the edge cases...